### DISCUSSION TITLE
pip install monailabel fails due to openslide-python error

### DISCUSSION THREAD

**Discussion by SenhorUnk** [Project-MONAI/MONAILabel#1205] [Q&A] [answered]
I had MONAILabel successfully installed on my own computer and another machine. Now I am trying to install Monailabel on an EC2 instance and the installation is not finishing due to an openslide error as shown below. I tried some suggestions from the internet but still no success. 

I would be really happy about help. 
I am running on Ubuntu 22.04.1 LTS

![image](https://user-images.githubusercontent.com/111501065/207113231-beb4c436-40a6-4e3f-ae05-f289411b5fae.png)

**Comment by diazandr3s** [comment, marked as answer]
Hi @SenhorUnk,

It seems there is an issue with openslide.
Please try to install it first and see how that goes. Here are the instructions: https://github.com/Project-MONAI/MONAILabel/issues/683#issuecomment-1082900491

Hope this helps,

**Reply by SenhorUnk** [reply]
Thanks for the quick response! 

Yesterday I was not using an virtual environment. I created one today using "conda create --name <env-name> python=3.8" which did the trick on my side. I didn't have to follow the steps in the instruction, that you have linked. Maybe that helps other people as well :)

**Comment by jakeacquadro** [comment]
For everyone arriving here with the same issue using python 3.11, there is currently no stable wheel for openslide-python: https://github.com/Project-MONAI/MONAILabel/issues/1569.
You will have to use python 3.10 for now.
